### ISSUE TITLE
I need to install the application and start a process with the cmd as administrator but with only one time the UAC popup?

### ISSUE THREAD

**Issue by ufukdeveloper** [oleg-shilo/wixsharp#1158]
Following problem: I want to start the cmd as administrator in C# in the InstallDir Dialog with a process to get the UAC before the installation:

```
var process = new Process();
var ps= new ProcessStartInfo();
ps.CreateNoWindow = true; 
ps.FileName = @"cmd.exe";
ps.Verb = "runas"; 
process.Start();
```

After i confirm the UAC, the installation starts. After the installation i want to start the same process but this time with arguments and without the UAC. Is that possible? 

Target is that i get a exe with the installation. I need the exe to run the arguments in the cmd. But the UAC should pop up before the exe will be appear. (the installation should happened after the uac, and the arguments after installation). PS: I need the administrator priviliges to run the arguments.

Is there any solution? Or another solution for the problem? I need to install the application and start a process with the cmd as administrator but want only one time the UAC popup. The uac must be up befor the process dialog.

**Comment by oleg-shilo** [comment]
Your msi session will pop elevation prompt just before starting the installation. This is expected behaviour. 
After the confirmation, your install will start but the session itself is kinda unelevated as any regular custom action will be able unelelevated. This is the best place to start the process if you want it to be unelevated.

Though if you want to start it elevated then you ned a deferred custom action. I suggest you use the one that is already provision for you by WixSharp runtime -  `project.AfterInstall += ...`
